plugins/aea-test-autonomy/aea_test_autonomy/base_test_classes/contracts.py

```diff
@@ -131,6 +135,12 @@ class BaseHardhatAMMContractTest(BaseContractTest, HardHatAMMBaseTest):
     """Base test case for testing AMM contracts on Hardhat."""
 
 
+class BaseRegistriesContractsTest(BaseContractTest, RegistriesBaseTest):
```

Comment:
This is not allowed here! Violation of separation of concerns

Reply:
I don't understand, the `BaseRegistriesContractsTest` is a class for testing the contract so shouldn't it belong in the base contract test classes module?